fms/__main__.py

```diff
@@ -1,120 +1,121 @@
+import argparse
+import sys
 
-import sys, optparse
-from utils import TypeEnforcer as te
 from happi import Client
-from happi.item import OphydItem
-from containers import FMSRaritanItem, FMSBeckhoffItem, FMSSRCItem
-from typing import List
+from happi.errors import SearchError
 
-fms_happi_database = "fms_test.json"
-def validate():
-    client = Client(path=fms_happi_database)
-    print(type(client.validate()))
+from .add_sensor import add_sensor
+from .check_topology import check_topology
+from .happi.containers import FMSRaritanItem
+
+fms_happi_database = "/cds/home/n/nrw/fms/db.json"
```

Comment:
Offline we discussed making this configurable via environment variable or otherwise

Reply:
moved database to ` /reg/g/pcds/pyps/apps/fms/db.json` and read from environment var.